package.json

```diff
@@ -75,7 +75,7 @@
     "babel-loader": "^7.1.2",
     "babel-plugin-transform-class-properties": "^6.24.1",
     "babel-plugin-transform-object-rest-spread": "^6.26.0",
-    "babel-preset-env": "^1.6.1",
+    "babel-preset-env": "1.7.0",
```

Comment:
remove this if not required.

Reply:
it was already from the starting just updated the package